UnityProject/Assets/Scripts/Systems/Chemistry/Effects/Item.cs

```diff
@@ -10,9 +10,17 @@ public class Item : Chemistry.Effect
 		private MixingBowl senderInfo;
 		private Vector3Int senderPosition;
 		public GameObject spawnItem;
+
+		public int numberToSpawn = -1;
+
 		public override void Apply(MonoBehaviour sender, float amount)
 		{
 			amount = (int)Math.Floor(amount);
+			if (numberToSpawn != -1)
```

Comment:
why not just make the default 1 then

Reply:
bigger reaction will have bigger number

Reply:
well what would happen if you only want to spawn one no matter the size of the reaction?